force-app/main/default/lwc/lightDomHello/lightDomHello.html

```diff
@@ -0,0 +1,10 @@
+<template lwc:render-mode="light">
+    <lightning-card title="LightDomHello" icon-name="custom:custom5">
+        <div class="slds-var-m-around_medium">Hello from Light DOM!</div>
```

Comment:
Can you sync with @adityanaag3 and align the recipes with LWC Recipes OSS?

Reply:
Hi @pozil, Changes Done! Thank you!